Release Notes/2018/2018-12-11-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,41 @@
+{{{
+"title": "Cloud Platform - Release Notes: December 11, 2018",
+"date": "12-11-2018",
+"author": "David Gardner",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### Enhancements (7)
+
+#### [Cloud Application Manager](https://www.ctl.io/cloud-application-manager/)
+
+#### [Application Lifecycle Management](https://www.ctl.io/cloud-application-manager/application-lifecycle-management/)
+
+**Support for Google Compute Deployment Manager**
+
+[Application Lifecycle Management](https://www.ctl.io/cloud-application-manager/application-lifecycle-management/) now supports a new type of template box based on Google Deployment Manager Templates. Now you can create this new type of template box and add multiple Google Deployment templates files in YAML format to it to compose your box. Similar to the current support for AWS Cloud Formation, Azure ARM and Terraform templates, you can also define variables through Cloud Application Manager user interface, and then reference them from any Google Deployment Manager template file in the instance, with the usual Jinja format. For more information, please refer to [https://www.ctl.io/knowledge-base/cloud-application-manager/automating-deployments/google-deployment-manager/](https://www.ctl.io/knowledge-base/cloud-application-manager/automating-deployments/google-deployment-manager/)
```

Comment:
Links to other KB articles should use the relative path to the .md file

Reply:
Hi Jim, just saw this - I'll address these issues. and resubmit the PR

Reply:
ok. fixed.